support-config/src/main/scala/com/gu/support/config/StripeConfig.scala

```diff
@@ -11,8 +13,19 @@ case class StripeConfig(
     unitedStatesAccount: StripeAccountConfig,
     version: Option[String],
 ) {
+  val secretForPublic: Map[StripePublicKey, (StripeAccountConfig, PaymentGateway)] =
+    List(
+      (defaultAccount, StripeGatewayPaymentIntentsDefault),
+      (australiaAccount, StripeGatewayPaymentIntentsAUD),
+      (unitedStatesAccount, StripeGatewayPaymentIntentsDefault), // US currently uses default account for recurring
+    ).map { config =>
+      config._1.publicKey -> config
+    }.toMap
```

Comment:
more DRY - reduces the chance of a copy paste error? 😂 
(although I do take your point!  I'll revisit it and try to make a simpler option)
![image](https://github.com/guardian/support-frontend/assets/7304387/b4ab302f-4516-49e9-a98f-4618c226080e)

Reply:
thanks I've basically done your suggestion in terms of not adding unnecessary logic, although I also realised that we only use the secret key in the StripeService, so I only return that instead.  I did also wrap it in a type rather than passing it around as a String.